content/csharp/getting-started/linq.md

```diff
@@ -145,24 +566,139 @@ There are extension methods that allow you to determine if or how many items wil
+        int peopleOverTwentyFive = people.Count(x => x.Age > 25);
+        Console.WriteLine(peopleOverTwentyFive);
+    }
+
+    public static List<Person> GenerateListOfPeople()
+    {
+        var people = new List<Person>();
+
+        people.Add(new Person { FirstName = "Eric", LastName = "Fleming", Occupation = "Dev", Age = 24 });
+        people.Add(new Person { FirstName = "Steve", LastName = "Smith", Occupation = "Manager", Age = 40 });
+        people.Add(new Person { FirstName = "Brendan", LastName = "Enrick", Occupation = "Dev", Age = 30 });
+        people.Add(new Person { FirstName = "Jane", LastName = "Doe", Occupation = "Dev", Age = 35 });
+        people.Add(new Person { FirstName = "Samantha", LastName = "Jones", Occupation = "Dev", Age = 24 });
+
+        return people;
+    }
+}
+
+public class Person
+{
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string Occupation { get; set; }
+    public int Age { get; set; }
+}
+```
 
 ###Any
 The `Any` extension method checks a set of data and return a boolean value indicating whether any items in the collection match your predicate. This is commonly used when checking if a list has any elements before performing some other action on the list.
 
-```c#
+```{.snippet}
 bool thereArePeople = people.Any(); //This will return true
-bool thereArePeople = emptyList.Any(); //This will return false
+bool thereAreNoPeople = emptyList.Any(); //This will return false
 ```
+```{.REPL}
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var people = GenerateListOfPeople();
+        var emptyList = new List<Person>();
 
+        bool thereArePeople = people.Any();
+        Console.WriteLine(thereArePeople);
+        bool thereAreNoPeople = emptyList.Any();
+        Console.WriteLine(thereAreNoPeople);
+    }
+
+    public static List<Person> GenerateListOfPeople()
+    {
+        var people = new List<Person>();
+
+        people.Add(new Person { FirstName = "Eric", LastName = "Fleming", Occupation = "Dev", Age = 24 });
+        people.Add(new Person { FirstName = "Steve", LastName = "Smith", Occupation = "Manager", Age = 40 });
+        people.Add(new Person { FirstName = "Brendan", LastName = "Enrick", Occupation = "Dev", Age = 30 });
+        people.Add(new Person { FirstName = "Jane", LastName = "Doe", Occupation = "Dev", Age = 35 });
+        people.Add(new Person { FirstName = "Samantha", LastName = "Jones", Occupation = "Dev", Age = 24 });
+
+        return people;
+    }
+}
+
+public class Person
+{
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string Occupation { get; set; }
+    public int Age { get; set; }
+}
+```
+
+###Tip {.tip .newLanguage}
 Using `Any` is a clearer way of checking if elements exist than checking if the count of a list is greater than 0 as shown in this example:
```

Comment:
This seems like a tip for everyone, not just new-to-programming people. I'd use this instead:
### Note { .note }
> Note content goes here.